migrations/mysql/20170219200159_add_foreign_key/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE fk_inits (
+    id INTEGER PRIMARY KEY
+);
+
+CREATE TABLE fk_tests (
+    id INTEGER PRIMARY KEY,
+    fk_id INTEGER NOT NULL REFERENCES fk_inits(id)
```

Comment:
MySQL accepts this as valid syntax (since it's part of the SQL standard), but it's ignored. The proper way to create a foreign key in MySQL is

```
fk_id INTEGER NOT NULL,
FOREIGN KEY (fk_id) REFERENCES fk_inits (id)
```